### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by SheldonGomel**
TODO: I only calculate the total number of seats for each type of carriage. When we have an array of occupied places, we will need to subtract this from the total number.

### CODE REVIEW

File: train-A-app/src/app/app.config.ts [hunk 1]
```diff
@@ -25,9 +33,13 @@ export const appConfig: ApplicationConfig = {
     provideStore({ roleState: rolesReducer.reducer }),
     provideState(stationFeature),
     provideState(carriageFeature),
+    provideState(searchFeature),
+    provideEffects([CarriageEffects]),
+    provideEffects([TripEffects]),
```
Comment (Mariia22) [inline train-A-app/src/app/app.config.ts]: We have a little mess here. All effects should be in one array.

File: train-A-app/src/app/home/components/trip-stations/trip-stations.component.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+})
+export class TripStationsComponent {
+  durations: string[] = [];
+
+  constructor(
+    public dialogRef: MatDialogRef<void>,
+    @Inject(MAT_DIALOG_DATA)
+    public data: { path: { stations: number[]; id: number }; schedule: Segment[] },
+  ) {
+    // TODO: get city names by id
```
Comment (Mariia22) [inline train-A-app/src/app/home/components/trip-stations/trip-stations.component.ts]: Is this comment necessary?

**Review by Mariia22 — APPROVED** [review]

File: train-A-app/src/app/home/components/trip-stations/trip-stations.component.ts (hunk 2 above)
Comment (SheldonGomel) [reply train-A-app/src/app/home/components/trip-stations/trip-stations.component.ts]: I totally forgot about it. I have to implement it.

File: train-A-app/src/app/app.config.ts (hunk 1 above)
Comment (SheldonGomel) [reply train-A-app/src/app/app.config.ts]: I added it to PR.

**Pull request merged**